link-grammar/prepare/build-disjuncts.c

```diff
@@ -295,7 +295,8 @@ build_disjunct(Sentence sent, Clause * cl, const char * string,
 
 		/* XXX add_category() starts category strings by ' '.
 		 * FIXME Replace it by a better indication. */
-		if (!IS_GENERATION(sent->dict) || (' ' != string[0]))
+		if (opts->use_sat_solver ||
```

Comment:
This needs to be wrapped with `#ifdef USE_SAT_SOLVER` -- that's why the ci tests are failing.